features/video-transformation/resize-crop-and-other-common-video-transformations.md

```diff
@@ -168,19 +168,43 @@ Notice that the height is 200px as requested, but the width is more than 200px.
 
 ![](<../../.gitbook/assets/image (39).png>)
 
-### Maintain ratio crop strategy
+### Maintain ratio crop strategy - (c-maintain_ratio)
 
-This is the default crop strategy. If nothing is specified in the URL, this strategy gets applied automatically. In this strategy, the output video's dimension (height and width) is the same as requested, and the aspect ratio is preserved. This is accomplished by resizing the video to the requested dimension and then cropping extra parts to get desired height & width.
+This is the default crop strategy. If no crop(`c`) or crop_mode(`cm`) is specified in the URL, `c-maintain_ratio` gets applied automatically.
+
+In this strategy, the output video's dimension (height and width) is the same as requested, and the aspect ratio is preserved. This is accomplished by resizing the video to the requested dimension and then cropping extra parts to get desired height & width.
 
 {% hint style="info" %}
 By default, ImageKit.io crops the video from the center but you can change this using the [focus parameter](../image-transformations/resize-crop-and-other-transformations.md#focus-fo).
 {% endhint %}
 
-URL - [https://ik.imagekit.io/demo/sample-video.mp4?tr=w-400,h-200](https://ik.imagekit.io/demo/sample-video.mp4?tr=w-400,h-200)\
-\
-Notice that the video's dimension matches 400x200 but the content is cropped from all edges i.e by default ImageKit will extract the video from the center. You can change this behaviour using the [focus parameter](resize-crop-and-other-common-video-transformations.md#focus-fo).
+{% tabs %}
+{% tab title="Original" %}
+URL - [https://ik.imagekit.io/demo/sample-video.mp4?tr=orig](https://ik.imagekit.io/demo/sample-video.mp4?tr=orig)
+
+![c-maintain_ratio,fo-center](../../.gitbook/assets/screenshot-sample-video-orig.png)
+{% endtab %}
+
+{% tab title="bg=w-200,h-300,c-maintain_ratio" %}
+URL - [https://ik.imagekit.io/demo/sample-video.mp4?tr=w-200,h-300,c-maintain_ratio](https://ik.imagekit.io/demo/sample-video.mp4?tr=w-400,h-300,c-maintain_ratio)
+
+This URL is equivalent to c-maintain_ratio [https://ik.imagekit.io/demo/sample-video.mp4?tr=w-200,h-300](https://ik.imagekit.io/demo/sample-video.mp4?tr=w-200,h-300) as `maintain_ratio` is the default crop strategy.
```

Comment:
Phrasing is a bit incorrect. We should say that `c-maintain_ratio` is applied by default and `https://ik.imagekit.io/demo/sample-video.mp4?tr=w-200,h-300` is equivalent to `https://ik.imagekit.io/demo/sample-video.mp4?tr=w-200,h-300,c-maintain_ratio` to be explicit in what we are trying to convey.

Also, if this example is here, why is the `Original` tab needed?

Reply:
Original input is required to show how this crop mechanism works. It's done similar for images too <https://docs.imagekit.io/features/image-transformations/resize-crop-and-other-transformations#maintain-ratio-crop-strategy-c-maintain_ratio>

Reply:
Updated the sentence to:

[`w-400,h-300,c-maintain_ratio`](https://ik.imagekit.io/demo/sample-video.mp4?tr=w-400,h-300,c-maintain_ratio) is equivalent to [`w-200,h-300`](https://ik.imagekit.io/demo/sample-video.mp4?tr=w-400,h-300) because `maintain_ratio` is the default crop strategy.